nimare/tests/test_meta_mkda.py

```diff
@@ -131,3 +148,20 @@ def test_analytic_empirical_convergence(testdata_cbma_full):
     # Correlation must be near unity and mean difference should be tiny
     assert np.corrcoef(p_analytical, p_empirical)[0, 1] > 0.98
     assert (p_analytical - p_empirical).mean() < 1e-3
+
+
+def test_kkda_analytic_empirical_convergence(testdata_cbma_full):
+    est_a = KDA(null_method="analytic")
+    n_iter = 10000
+    est_e = MKDADensity(null_method="empirical", n_iter=n_iter)
```

Comment:
I missed the second estimator, but the test is passing, which worries me a little. Especially since it's using `testdata_cbma_full`, which should have multiple foci per study.

Reply:
Perhaps none of the foci are near enough to one another for the KDA/MKDA distinction to matter?

Reply:
I'm sure it's not a big deal, so I'm going to resolve this.